### ISSUE TITLE
can someone explain what these too me pls

### ISSUE THREAD

**Issue by Franzferdinan51** [turtlecoin/turtle-pool#44]
2018-10-10 14:52:43 [api] Stat collection finished: 1 ms redis, 5 ms daemon
2018-10-10 14:52:43 [api] Error deflating data: null
2018-10-10 14:52:43 [api] Broadcasting to 0 visitors and 0 address lookups
2018-10-10 14:52:43 [api] Redis error occurred: null

(node:7367) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

im seeing it upon starting the pool not sure what they mean honestly the pool continues to go on it doesnt stop or anything though i havent tested mining on it either so idk mostly curious so that i can fix them

**Comment by zpalmtree** [comment]
It's nothing to worry about. Essentially it's saying - in the future, you shouldn't use this function, `Buffer()`, and you should instead use the following functions. The code will continue to work, but perhaps the function is badly designed so if you misuse it, you leak memory, or something like that. Functions get deprecated now and again when mistakes have been made implementing them, but they can't break code for people already using them.

tl;dr: It doesn't matter,

**Comment by Franzferdinan51** [comment]
ty yeah i managed to get a mining test and it affects nothing :) thank you very much sir!

**Comment by brandonlehmann** [comment]
Question: are you running the latest and greatest copy here? I replaced the standard buffer with one that is safe and doesn't generate that warning.
